includes/data-stores/class-wc-admin-reports-customers-data-store.php

```diff
@@ -505,6 +506,43 @@ public static function get_or_create_customer_from_order( $order ) {
 		return $result ? $customer_id : false;
 	}
 
+	/**
+	 * Try to get a customer name from user profile or order information.
+	 *
+	 * @param int      $user_id User ID.
+	 * @param WC_Order $order Order made by customer.
+	 * @return array
+	 */
+	public static function get_customer_name( $user_id = 0, $order = null ) {
```

Comment:
Can we eliminate the `$user_id` parameter and just use `$order->get_user_id()` internally?

Reply:
We can't quite get rid of `$user_id` here since the it's possible there is no last order made (customer created without order or customer created and order was deleted).  And inversely, `$user_id` could be `0` when there's an order made by a guest.